apps/verticalhome/js/pinned_apps_navigation.js

```diff
@@ -13,274 +13,375 @@
       return;
     }
 
+    //this.elements[2].classList.add('selected');
+    this.elements =  document.getElementsByClassName('pinned-app-item');
     this.elemList = list;
     window.removeEventListener('keydown', this);
```

Comment:
"If the keydown handler has already been added, adding it again will be ignored, so I think it is safe to just leave out the removeEventListener()" :copyright: **(David)**